prowler/providers/aws/services/dms/dms_service.py

```diff
@@ -49,6 +52,68 @@ def _describe_replication_instances(self, regional_client):
                 f"{regional_client.region} -- {error.__class__.__name__}[{error.__traceback__.tb_lineno}]: {error}"
             )
 
+    def _describe_endpoints(self, regional_client):
+        logger.info("DMS - Describing DMS Endpoints...")
+        try:
+            describe_endpoints_paginator = regional_client.get_paginator(
+                "describe_endpoints"
+            )
+            for page in describe_endpoints_paginator.paginate():
+                for endpoint in page["Endpoints"]:
+                    arn = endpoint["EndpointArn"]
+                    if not self.audit_resources or (
+                        is_resource_filtered(arn, self.audit_resources)
+                    ):
+                        self.endpoints[arn] = Endpoint(
+                            id=endpoint["EndpointIdentifier"],
+                            ssl_mode=endpoint.get("SslMode", False)
+                        )
+        except Exception as error:
+            logger.error(
+                f"{regional_client.region} -- {error.__class__.__name__}[{error.__traceback__.tb_lineno}]: {error}"
+            )
+
+
+    def _describe_data_providers(self, regional_client):
```

Comment:
Hi @LefterisXefteris , why did you create `_describe_data_providers` function?

Reply:
Moreover, could you give us access to the branch so we can help you better with the changes? Thanks!